h2spatial-ext/src/test/java/org/h2gis/h2spatialext/function/spatial/raster/RasterFunctionTest.java

```diff
@@ -1396,4 +1397,40 @@ public void testST_FillSinksChezine() throws Exception {
             }
         }
     }
+    
+    @Test
+    public void testST_DumpAsPolygons1() throws SQLException, IOException {        
+        int width = 5;
+        int height = 4;
+        final double noData = 0;
+        final float pixelSize = 100;
+
+        int[] pixels = new int[]{
+            0, 0, 0, 0, 0,
+            0, 1, 0, 0, 0,
+            0, 1, 1, 1, 0,
+            0, 0, 0, 0, 0
+        };
+                
+        // Create image from int array
+        RenderedImage image = imageFromArray(pixels, width, height);
+        st.execute("drop table if exists test, vectorize;");
+        st.execute("create table test(id identity, the_raster raster)");
+        // Create table with test image
+        PreparedStatement ps = connection.prepareStatement("INSERT INTO TEST(the_raster) " + "values(?)");
+        ps.setBinaryStream(1, GeoRasterRenderedImage.create(image, pixelSize, -pixelSize, 0, height, 0, 0, 27572, noData)
+                .asWKBRaster());
+        ps.execute();
+        ps.close();
+
+        st.execute("create table vectorize as select ST_DumpAsPolygons(the_raster) FROM test");
```

Comment:
DumpAsPolygons is a table function.

This **may** work following your signature.
```sql
create table vectorize as select * from ST_DumpAsPolygons((SELECT the_raster from test limit 1));
```